src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
+
+    subtract: function (valueResult) {
+      return function (value) {
+        return valueResult - value;
+      };
+    },
+
+    multiply: function (valueResult) {
+      return function (value) {
+        return valueResult * value;
+      };
+    },
+
+    divide: function (valueResult) {
+      return function (value) {
+        return valueResult / value;
```

Comment:
It is necessary to add a check for 0, because 0 can be divided by according to the rules of mathematics

Reply:
It wasn't in the tests, but it's helpful to do. thank you